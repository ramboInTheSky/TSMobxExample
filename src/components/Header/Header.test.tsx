import React from 'react'
import { shallow } from 'enzyme'

import { Header } from '.'

describe('<Header />', () => {
	it('renders correctly', () => {
		expect(shallow(<Header />)).toMatchSnapshot()
	})

	// it('passes on props correctly', () => {
	// 	expect(shallow(<Header foo="bar">Hello</Header>)).toMatchSnapshot()
	// })

	// it('Does not re-render on children props change', () => {
	// 	const el: any = shallow(<Header foo="bar" />)
	// 	el.setProps({ children: 'Goodbye' })
	// 	el.update()
	// 	expect(el).toMatchSnapshot()
	// })
})
