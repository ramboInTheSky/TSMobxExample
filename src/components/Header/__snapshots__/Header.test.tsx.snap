// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<Boundary>
  <Styled(div)>
    <Styled(img)
      alt="The Collective"
      src="logo_dark_vertical.svg"
    />
    <Styled(div)>
      ROOST
    </Styled(div)>
    <div
      id="anchor-000"
    >
      <Styled(WithStyles(IconButton))
        aria-haspopup="true"
        aria-owns="Settings"
        onClick={[Function]}
      >
        <pure(SettingsIcon) />
      </Styled(WithStyles(IconButton))>
      <WithStyles(Menu)
        anchorEl={null}
        aria-label="Settings"
        id="Settings"
        onClose={[Function]}
        open={false}
        title="Settings"
      >
        <WithStyles(MenuItem)
          onClick={[Function]}
        >
          UI version: 
          0.1.1
        </WithStyles(MenuItem)>
        <WithStyles(MenuItem)
          onClick={[Function]}
        >
          Build number: 1.0.0
        </WithStyles(MenuItem)>
      </WithStyles(Menu)>
    </div>
  </Styled(div)>
</Boundary>
`;
